apps/overige-objecten-api/src/server.ts

```diff
@@ -46,7 +48,12 @@ const corsOption: CorsOptions = {
 };
 const apiSpec = path.join(__dirname, './docs/openapi.yaml');
 const app = express();
+// Multer file upload middleware.
+// The order is important, so this should be before the express.json() middleware to parse the file.
+const upload = multer({ dest: 'tmp/uploads/' });
```

Comment:
Deze temporary file moet ook weer verwijderd worden.

Reply:
Het is opgelost door gebruik te maken van await `fs.promises.unlink(filePath)` in plaats van `fs.unlinkSync(filePath)`.

https://github.com/frameless/strapi/blob/enable-to-import-vac-as-csv/apps/overige-objecten-api/src/controllers/import/index.ts#L50